app/models/manageiq/providers/hawkular/middleware_manager/event_parser.rb

```diff
@@ -0,0 +1,14 @@
+module ManageIQ::Providers::Hawkular::MiddlewareManager::EventParser
+  def self.event_to_hash(event, ems_id = nil)
+    $mw_log.debug "ems_id: [#{ems_id}] event: [#{event.inspect}]"
+    {
+      :event_type => event[:event_type],
+      :source     => 'HAWKULAR',
+      :timestamp  => event[:timestamp],
+      :message    => event[:message],
+      :ems_ref    => event[:resource], # not sure about this, possible link from event to resource?
```

Comment:
ems_ref is a link to the `ExtManagementSystem`
I dont see any other provider filling this hash key. Its mostly `:host_ems_ref` or `:vm_ems_ref` 
And I'm not even sure how this field is used on the other side of the Event system.

How did you come to include this field in the first place?